packages/table/src/cell/editableCell.tsx

```diff
@@ -44,27 +52,37 @@ export class EditableCell extends React.Component<IEditableCellProps, {}> {
     private cellElement: HTMLElement;
 
     public render() {
-        const { className, value, intent, onCancel, onChange, onConfirm, style, tooltip } = this.props;
+        const { className, value, intent, loading, onCancel, onChange, onConfirm, style, tooltip } = this.props;
+
+        const classes = classNames(
+            CELL_CLASSNAME,
+            Classes.intentClass(intent),
+            { [Classes.LOADING]: loading },
+            className,
+        );
+
         return (
             <div
-                className={classNames(className, Classes.intentClass(intent), "bp-table-cell")}
+                className={classes}
+                ref={this.handleCellRef}
                 style={style}
                 title={tooltip}
-                ref={this.handleCellRef}
             >
-                <Draggable onDoubleClick={this.handleCellDoubleClick}>
-                    <EditableText
-                        className={"bp-table-editable-name"}
-                        defaultValue={value}
-                        intent={intent}
-                        minWidth={null}
-                        onCancel={onCancel}
-                        onChange={onChange}
-                        onConfirm={onConfirm}
-                        placeholder=""
-                        selectAllOnFocus={true}
-                    />
-                </Draggable>
+                <LoadableContent loading={loading} variableLength={true}>
```

Comment:
would be cool to refactor to compose `Cell` in here instead of duplicating most of the logic, but we need #427 to merge first for `truncated` prop.